source/ShipInfoPanel.cpp

```diff
@@ -118,8 +118,8 @@ void ShipInfoPanel::Draw()
 
 	// Draw the interface.
 	const Interface *infoPanelUi = GameData::Interfaces().Get("info panel");
-	infoPanelUi->Draw(interfaceInfo, this);
-	int infoPanelLine = 0;
+	infoPanelUi->Draw(info, this);
+	// int infoPanelLine = 0; Removed by the 20241125conflictupdate not sure if it is upstream or Delta's.
```

Comment:
```suggestion
	// int infoPanelLine = 0; Removed by the 2024-11-25 conflict update. Not sure if it is upstream or Delta's.
```

Reply:
That word is directly referencing the name of my branch title, so I would prefer to keep it as the branch title and not spacing it out as a sentence. It'll also be deleted before this is merged.

Reply:
If it'll be deleted then feel free to ignore. Just pointing it out.